From 9c210763508ed5490bbdc7aa7d2d2a205a60658d Mon Sep 17 00:00:00 2001
From: Paul Gortmaker <paul.gortmaker@windriver.com>
Date: Fri, 27 May 2011 07:08:41 -0400
Subject: pm_runtime.h: explicitly requires notifier.h

This file was getting notifier.h via device.h --> module.h but
the module.h inclusion is going away, so add notifier.h directly.

Signed-off-by: Paul Gortmaker <paul.gortmaker@windriver.com>
(cherry picked from commit 246359d37985000b8403487e46867c4eb610af72)

Signed-off-by: Simon Horman <horms@verge.net.au>
---
 include/linux/pm_runtime.h |    1 +
 1 file changed, 1 insertion(+)

diff --git a/include/linux/pm_runtime.h b/include/linux/pm_runtime.h
index 70b2840..d8d9036 100644
--- a/include/linux/pm_runtime.h
+++ b/include/linux/pm_runtime.h
@@ -10,6 +10,7 @@
 #define _LINUX_PM_RUNTIME_H
 
 #include <linux/device.h>
+#include <linux/notifier.h>
 #include <linux/pm.h>
 
 #include <linux/jiffies.h>
-- 
1.7.10.1.362.g242cab3

